pkgs/tools/backup/tsm-client/default.nix

```diff
@@ -0,0 +1,169 @@
+{ lib
+, stdenv
+, autoPatchelfHook
+, buildEnv
+, fetchurl
+, gnutar
```

Comment:
I believe this would be unnecessary if you would depend on `stdenv` instead of `stdenv.cc.cc` later. This way, you make sure to use the tar package from `stdenv`. So, unless you need a tar feature, that only `gnutar` has, this would be more general.

I am not sure about the impact on `autoPatchelfHook`, though...